### PR TITLE
Make compatible with v3

### PR DESCRIPTION
**Pull request by sunloverz**

### CODE REVIEW

File: lib/delayed/backend/actions.rb [hunk 1]
```diff
@@ -31,7 +31,11 @@ def db_time_now
 
         def find_available(worker_name, limit = 5, max_run_time = Worker.max_run_time)
           Delayed::Worker.available_priorities.each do |priority|
-            message = ironmq.queue(queue_name(priority)).get
+            begin
+              message = ironmq.queue(queue_name(priority)).get
```
Comment (thousandsofthem) [inline lib/delayed/backend/actions.rb]: message defined within begin/rescue block and could not be available outside (in case of exception e.g.)

**Comment by thousandsofthem** [comment]
@sunloverz please add some info about setting `host` along with project_id/token to readme

**Comment by sunloverz** [comment]
Done, but need to implement some extra requirements, an error queue for failed jobs, retry etc.

**Comment by thousandsofthem** [comment]
Looks good for me, can merge

**Comment by sunloverz** [comment]
@thousandsofthem could you please review? Added some changes

**Comment by thousandsofthem** [comment]
Code looks good :+1:

**Pull request merged**